<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCCurve</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: black</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="logButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: red</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: blue</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow</string>
    </property>
    <property name="text">
     <string>Registrar Usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Nombre de Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="passLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLabel" name="confirmLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Confirmar Contraseña:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: green</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow</string>
    </property>
    <property name="text">
     <string>Iniciar Sesión</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>289</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>289</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>289</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white</string>
    </property>
   </widget>
   <widget class="QLabel" name="warningLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leftLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>400</width>
      <height>246</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rightLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>400</width>
      <height>246</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dccLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>62</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>DCCurve</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
